deployment/permissions/service.livestream.ini

```diff
@@ -130,3 +130,12 @@ permissionItem19.service = livestream
 permissionItem19.action = createRecordedEntry
 permissionItem19.partnerId = 0
 permissionItem19.permissions = -5>MEDIA_SERVER_BASE, -5>PARTNER_-5_GROUP_*_PERMISSION,MEDIA_SERVER_PARTNER_LEVEL
+
+permissionItem20.service = livestream
+permissionItem20.action = archive
+permissionItem20.partnerId = -1
+permissionItem20.param3 =
+permissionItem20.param4 =
+permissionItem20.param5 =
+permissionItem20.tags =
+permissionItem20.permissions = BATCH_BASE, PARTNER_-1_GROUP_*_PERMISSION
```

Comment:
Who is supposed to call this api? is only the batch able to call it? does not make sense

Reply:
Currently yes
in the future we want to be able to use this API as part of the webcasting app.